packages/editor/src/components/preferences-modal/index.js

```diff
@@ -152,6 +154,18 @@ export default function EditorPreferencesModal( { extraSections = {} } ) {
 											'Enable pre-publish checks'
 										) }
 									/>
+									{ starterPatterns?.length && (
+										<PreferenceToggleControl
```

Comment:
I added this under "Publishing" for no real reason. Is there a better category?

Reply:
I'd possibly lean ever so slightly toward Interface as it's about what we're showing to the user in the UI rather than publishing per se... but I don't feel strongly about it!